services/website-api/src/routes/routes.ts

```diff
@@ -340,6 +344,53 @@ export function RegisterRoutes(app: express.Router) {
 
     // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
 
+    function authenticateMiddleware(security: TsoaRoute.Security[] = []) {
```

Comment:
I know this is auto generated but shouldnt the middleware in their own file?

Reply:
middleware is in its own file, just gets used in the routes since tsoa generates it that way